src/javascript/app_2/App/Components/Elements/Tooltip2/tooltip-2.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes     from 'prop-types';
+import React         from 'react';
+import TooltipBubble from './tooltip-bubble.jsx';
```

Comment:
if i am not mistaken it named `TooltipBalloon` instead ?

Reply:
It can be used Interchangeably but I like bubble better because it resembles a speech bubble.